controllers/region.js

```diff
@@ -290,9 +290,9 @@ export const getObjRegionCids = obj => {
  * Return populated array of regions for transferred object
  * @param obj Object (photo, comment etc.)
  * @param fields Selected region fields. Array, but in case of 'fromDb' - object
- * @param [fromDb] Select from db, not just from cache (in cahce not all fields presented)
+ * @param fromDb Select from db, not just from cache (in cahce not all fields presented)
```

Comment:
fromDb is still optional, right?

Reply:
Yes, correct (I had to check JSDoc, I took it as array first).

Reply:
BTW Correct syntax for destructing params https://jsdoc.app/tags-param.html#parameters-with-properties Anyway, leaving this as is for now (we may add JSDoc liniting one day).

Reply:
You are right, feel free to change that. When these funtion was written, jsdoc didn't know about destructuring yet.